src/factory.js

```diff
@@ -118,6 +121,19 @@ export class Factory {
     return note;
   }
 
+  GraceNote(noteStruct) {
+    const note = new GraceNote(noteStruct);
+    if (this.stave) note.setStave(this.stave);
+    note.setContext(this.context);
+    return note;
+  }
+
+  GraceNoteGroup(params) {
+    const group = new GraceNoteGroup(params.notes, params.slur);
```

Comment:
If it's unused, you can leave it for now. It's easy to add it if we need it without breaking users.